src/PowerShellEditorServices/Analysis/AnalysisService.cs

```diff
@@ -24,6 +24,47 @@ namespace Microsoft.PowerShell.EditorServices
     /// </summary>
     public class AnalysisService : IDisposable
     {
+        #region Static fields
+
+        /// <summary>
+        /// Defines the list of Script Analyzer rules to include by default if
+        /// no settings file is specified.
+        /// </summary>
+        private static readonly string[] s_includedRules = new string[]
```

Comment:
Don't know how everyone feels about this, but you can ditch the `new string[]` and just keep the braces for a field initialization.  e.g
```csharp
private static readonly string[] s_includedRules =
{
    "MyRules",
};